### DISCUSSION TITLE
UUID support for delete/transfer

### DISCUSSION THREAD

**Discussion by Victor75007** [GriefPrevention/GriefPrevention#551] [Ideas]
Hello

Can i delete someone's claim-lands with UUID ?
I use DBA plugin, so my players can change from cracked to premium/online character.
So these players get new UUID, and i can't transfer old claims to new UUID.
Or i can't delete old offline/cracked UUID's claims.

If there is no such option, can you add it ?

Sorry my english

Have a nice day!
Best regards,
Victor

**Comment by bigpresh** [comment]
You, as admin, could transfer them by standing in their claim and using the `/transferclaim` command - but I rather suspect you're looking for the option to automatically transfer over all their claims.  No such ability exists, and I don't think it should be a core function as it wouldn't be useful to many people - but someone could probably implement it as an addon.  It'd also be possible as a script, but I think you'd probably need to stop the server first so a script could replace their old UUID with the new one in each claim that belonged to them, then start back up, which would be a hassle.

Or you could make the players who are changing UUID do the work themselves, by having them connect using their old details, and trust their new account in each claim.  (Or, of course, just stop supporting pirated copies of Minecraft entirely :) )

**Comment by Jikoo** [comment]
I agree that OP's use case is not something I care for, never been a fan of offline support, but the request is not "make me an offline support system for players changing UUIDs," it is "allow identifying users by UUID instead of name in commands."

I think this is a good core feature, and would solve issues like the age-old issues with players not being found due to them having been offline for more than GP's allotted time. I couldn't find a specific issue for that but I know I've seen it in the past, admins trying to expire ancient claims and things like that.
